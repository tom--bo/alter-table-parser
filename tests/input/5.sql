alter table t1 rename index idx_a to idx_b;
