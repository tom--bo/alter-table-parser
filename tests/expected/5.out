Input Table Definition: (Please type ^d to end input) 
type = 21
successfully ended
